src/Playroom/Playroom.css.ts

```diff
@@ -7,11 +7,18 @@ import {
 import { sprinkles, colorPaletteVars } from './sprinkles.css';
 import { vars } from './vars.css';
 import { toolbarItemSize } from './ToolbarItem/ToolbarItem.css';
-import { toolbarItemCount, toolbarOpenSize } from './toolbarConstants';
+import {
+  ANIMATION_TIMEOUT,
+  toolbarItemCount,
+  toolbarOpenSize,
+} from './constants';
 
 export const MIN_HEIGHT = toolbarItemSize * toolbarItemCount;
 export const MIN_WIDTH = toolbarOpenSize + toolbarItemSize + 80;
 
+const MAX_HEIGHT = '90vh';
```

Comment:
Even though these values already existed, this should probably be reduced to `80vh`. At `90vh` the frames have already disappeared entirely.